File: 283-move-zeroes/move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int lastz=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]!=0){
                nums[lastz]=nums[i];
                lastz++;
            }
        }

        for(int i=lastz;i<nums.size();i++){
            nums[i]=0;
        }
    }
};

File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        last0=0
        for i in range(len(nums)):
            if nums[i]!=0:
                nums[last0]=nums[i]
                last0+=1
        

        for i in range(last0,len(nums)):
            nums[i]=0